modules/core/src/scenegraph/gltf-instantiator/gltf-material.js

```diff
@@ -252,6 +254,18 @@ class GLTFMaterialParser {
       this.parseTexture(material.emissiveTexture, 'u_EmissiveSampler', 'HAS_EMISSIVEMAP');
       this.uniforms.u_EmissiveFactor = material.emissiveFactor || [0, 0, 0];
     }
+    if (material.alphaMode === 'MASK') {
```

Comment:
* use switch statement (case 'MASK': case 'BLEND)
* Break out a helper function to handle the blend setup?
* Changing to `MASK` does not reset `parameters`?